table.cc

```diff
@@ -1747,7 +1764,7 @@ future<> table::snapshot(sstring name) {
                     auto rf = f.substr(sst->get_dir().size() + 1);
                     table_names.insert(std::move(rf));
                 }
-                return smp::submit_to(shard, [requester = engine().cpu_id(), jsondir = std::move(jsondir),
+                return smp::submit_to(shard, [requester = engine().cpu_id(), jsondir = std::move(jsondir),this,
```

Comment:
I only found one super-tiny style mistake - please put a space between the comma and 'this' - I can push the series after this.

Reply:
I have forced push an updated version with the missing space